test-swift-rust-integration.sh

```diff
@@ -11,17 +7,22 @@ THIS_DIR=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
 ROOT_DIR="$THIS_DIR"
 cd "$ROOT_DIR"
 
-cd SwiftRustIntegrationTestRunner
-# If project files don't exist before Xcode begins building we get something like:
-# error: Build input file cannot be found: '/path/to/Generated/SwiftBridgeCore.swift'
-# So.. here we create empty versions of the files that will get generated during the
-# build so that Xcode knows about them.
-# During the build process these will get overwritten with their real final contents.
-touch ./Generated/SwiftBridgeCore.{h,swift}
-mkdir -p ./Generated/swift-integration-tests
-touch ./Generated/swift-integration-tests/swift-integration-tests.{h,swift}
+# Compile the rust library used by the integration tests.
+cargo build --manifest-path crates/swift-integration-tests/Cargo.toml
+
+# Create a swift file with the generated swift code and an import for the
+# generated C code. This is necessary because SPM projects don't support
+# mixed-source - the C code has to be a separate SPM target.
```

Comment:
This is something that should be addressed upstream before this goes any farther. Basically, unlike, xcode, SPM doesn't allow mixed-language modules - you can't put `.swift` and `.h` files in the same target. Right now, the generation code assumes that all the swift and C code will coexist in the same namespace.

Reply:
#315. I think taking the approach I outlined there would address a lot of the issues here, for example the hardcoded output path in `build.rs`.